modules/gitops-release-notes-1-4-0.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-0_{context}"]
+= Release notes for {gitops-title} 1.4.0
+
+{gitops-title} 1.4.0 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="new-features-1-4-0_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.4.0:
+
+* Upgraded {gitops-title} Application Manager (kam) to version *0.0.41*. link:https://issues.redhat.com/browse/GITOPS-1669[GITOPS-1669]
+
+* Upgraded Argo CD to version *2.2.2*. link:https://issues.redhat.com/browse/GITOPS-1532[GITOPS-1532]
+
+* Upgraded Helm to version *3.7.1*. link:https://issues.redhat.com/browse/GITOPS-1530[GITOPS-1530]
+
+* This enhancement adds the health status of `DeploymentConfig` resources, `Route` resources, and Operators that you install using OLM to the Argo CD Dashboard and {product-title} web console. This enables you to better monitor the overall health status of your application. link:https://issues.redhat.com/browse/GITOPS-655[GITOPS-655], link:https://issues.redhat.com/browse/GITOPS-915[GITOPS-915], link:https://issues.redhat.com/browse/GITOPS-916[GITOPS-916], link:https://issues.redhat.com/browse/GITOPS-1110[GITOPS-1110]
```

Comment:
Shouldn't it be `These enhancements add the health status...` since you are referring to the multiple upgrades you mention above?

Reply:
Last sentence, `This enables you to better monitor...` I'm not sure what you refer to when you say `this` because you've mentioned a lot of things previously. 

Do you mean, `The Argo CD Dashboard and the OpenShift Container Platform web console both enable you to better monitor...`? 

If you're referring to everything you listed above, I think you could simply say, `All these enhancements enable you to better monitor...`

Reply:
> Shouldn't it be `These enhancements add the health status...` since you are referring to the multiple upgrades you mention above?

"Enhancement," here, refers to this singular list item, not the list items that precede it.

Reply:
Thanks. Updated.